ground/gcs/src/app/main.cpp

```diff
@@ -140,6 +140,34 @@ static inline QString msgCoreLoadFailure(const QString &why)
     return QCoreApplication::translate("Application", "Failed to load core: %1").arg(why);
 }
 
+static inline QStringList getPluginPaths()
+{
+    QStringList rc;
+    // Figure out root:  Up one from 'bin'
+    QDir rootDir = QApplication::applicationDirPath();
+    rootDir.cdUp();
+    const QString rootDirPath = rootDir.canonicalPath();
+    // 1) "plugins" (Win/Linux)
+    QString pluginPath = rootDirPath;
+    pluginPath += QLatin1Char('/');
+    pluginPath += QLatin1String(GCS_LIBRARY_BASENAME);
+    pluginPath += QLatin1Char('/');
+    pluginPath += QLatin1String("plugins");
+    rc.push_back(pluginPath);
+    // 2) "PlugIns" (OS X)
+    pluginPath = rootDirPath;
+    pluginPath += QLatin1Char('/');
+    pluginPath += QLatin1String("Plugins");
+    rc.push_back(pluginPath);
+    return rc;
+}
+
+#ifdef Q_OS_MAC
+#  define SHARE_PATH "/../Resources"
+#else
+#  define SHARE_PATH "/../share/dronin"
```

Comment:
Actually I have not tested this.. is this supposed to be /../share now?  Could this be why the welcome screen on windows is blank?